### ISSUE TITLE
vgremove command can "get stuck" waiting for prompt

### ISSUE THREAD

**Issue by phlogistonjohn** [heketi/heketi#1244]
### Kind of issue

Bug

### Observed behavior

Running a heketi command to add a device never completed without additional intervention.
Example:
```
oc exec heketi-7d88c9cd74-pp956 -- heketi-cli device add --node=87b0a2168e3c1d6ef1ce4ad5e6dc930b  --name=/dev/bob
```

### Expected/desired behavior

Command exits with success or failure within a "reasonable" time frame, regardless of the state of the system. Something like
```
$ oc exec heketi-7d88c9cd74-pp956 -- heketi-cli device add --node=87b0a2168e3c1d6ef1ce4ad5e6dc930b  --name=/dev/bob
Error: Unable to execute command on glusterfs-rzlk8:   Device /dev/bob not found.
command terminated with exit code 255
```

### Details on how to reproduce (minimal and precise)

The key to this error is getting lvm tools in a bad state. Thus it is somewhat difficult to give minimal steps. While trying to reproduce a different scenario I:

* Manually opened a file on a brick fs so that umount commands would fail
* Did a node replacement, migrating bricks off of the brick fs in question
* Used heketi-cli to delete the device which heketi allowed because it considered the device devoid of bricks  (see below)
* Ran heketi-cli device add with a dummy device name on the same node as the "bad" brick

Looking into the state of the system I can see that multiple lvm commands are running:
* pvcreate /dev/bob (x2, because I ran the command multiple times)
* vgremove vg_b934a3f13f3dca5601931b346fa9c42c

strace revealed that pvcreate is blocked on flock of an lvm lock file. strace of vgremoved revealed that it was reading from stdin. Running the same command manually prompts like:
```
# vgremove  vg_b934a3f13f3dca5601931b346fa9c42c
Do you really want to remove volume group "vg_b934a3f13f3dca5601931b346fa9c42c" containing 2 logical volumes? [y/n]: n
  Volume group "vg_b934a3f13f3dca5601931b346fa9c42c" not removed
```

### Information about the environment:

- Heketi version used (e.g. v6.0.0 or master): master

### Other useful information

We should take care to make sure we run our lvm commands in non-prompting mode. I think we can simply change the code in this case to `vgremove -qq` so that vgremove will not prompt.

**Comment by mykaul** [comment]
LVM is not very good at running commands in parallel, do we have some mechanism to try and avoid it?

**Comment by obnoxxx** [comment]
@mykaul writes:
> I thought (looking at https://github.com/heketi/heketi/blob/6ee7ab4e3b4ea55d9967dfafaf2847b47ac2dadc/executors/cmdexec/device.go#L76 )
> we should do:
> vgremove -f

Maybe, but this is unrelated to this issue at hand. (The problem and fix were different. ) We can open a new issue for this, if needed.

**Comment by obnoxxx** [comment]
@mykaul writes:
> LVM is not very good at running commands in parallel, do we have some mechanism to try and avoid it?

Not really, afaik...
